MonoZelda/MonoZeldaGame.cs

```diff
@@ -82,6 +84,13 @@ protected override void Update(GameTime gameTime)
         scene.Update(gameTime);
 
         base.Update(gameTime);
+        if (PlayerState.IsDead)
+        {
+            commandManager.Execute(CommandType.PlayerDeathCommand);
+            PlayerState.IsDead = false;
+            PlayerState.Initialize();
```

Comment:
Could these two lines be done in the PlayerDeathCommand instead to simplify the Game class?